docs/release-notes/1.9.0.asciidoc

```diff
@@ -8,6 +8,8 @@
 [float]
 === Breaking changes
 
+IMPORTANT: For OLM users that run with automatic upgrades enabled, it is advised to set `set-default-security-context` link:https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-operator-config.html[operator flag] explicitly before next ECK release. Next version of ECK will introduce a link:https://github.com/elastic/cloud-on-k8s/issues/5061[change] that could result in setting an invalid Pod Security Context to Elasticsearch Pods.
```

Comment:
Should we also add a note for the OpenShift OperatorHub users?

Reply:
Don't we include include OpenShift OperatorHub by mentioning OLM? (I thought this is the same mechanic)

Reply:
It requires the reader to know how OpenShift OperatorHub is implemented. Even if OLM is used behind the scenes, I think that it makes it obvious that OpenShift users should read this note carefully.